api/src/unraid-api/graph/resolvers/customization/customization.service.ts

```
@@ -0,0 +1,472 @@
+    private readonly activationJsonExtension = '.activationcode';
+
+    // Path properties - will be initialized in onModuleInit or from getters
+    private activationDir!: string;
+    private hasRunFirstBootSetup!: string;
+    private configFile!: string;
+    private caseModelCfg!: string;
+    private identCfg!: string;
+
+    private activationData: ActivationCode | null = null; // Store validated data here
+
+    constructor() {}
```

Biome (1.9.4)

[error] This constructor is unnecessary.

Unsafe fix: Remove the unnecessary constructor.


(lint/complexity/noUselessConstructor)